R/spatial_block_cv.R

```diff
@@ -85,6 +80,16 @@ spatial_block_cv <- function(data,
     )
   }
 
+  if (sf::st_is_longlat(data) && !sf::sf_use_s2()) {
+    rlang::abort(
+      c(
+        "`spatial_block_cv()` can only process geographic coordinates when using the s2 geometry library",
```

Comment:
This fell out while debugging CI -- sf will give many, many warnings due to returning inaccurate results when using geographic coordinates without s2, but for some reason those warnings didn't trigger anything in CI. The results are wrong enough that I think it's worth throwing an error